src/windows/enumerate.rs

```diff
@@ -287,7 +290,7 @@ impl PortDevice {
 
         let mut port_name_buffer = [0u16; MAX_PATH];
         let mut port_name_len = port_name_buffer.len() as DWORD;
```

Comment:
Don't we have to provide the length in terms of `u8` to `RegQueryValueExW` as for `RegEnumValueW` below?
```suggestion
        let mut port_name_len = port_name_buffer.len() * 2 as DWORD;
```

Reply:
fixed